src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+        },
+        "temperature (Celsius)": {
+            "title": "Temperature",
+            "type": "number",
+            "comment": "Temperature can only be specified in Celsius"
+        },
+        "post_treatment_time": {
+            "title": "Post-treatment time",
+            "description": "The time past the point when biosamples were removed from the treatment solution before being sampled or treated with the next treatment.",
+            "comment": "Use in conjunction with treatment and post_treatment_time_units to specify time elapsed post-treatment. post_treatment_time specification requires non-empty treatments list and post_treatment_time_units specification.",
+            "type": "string",
+            "pattern": "^(\\d+(\\.[1-9])?(\\-\\d+(\\.[1-9])?)?)|(unknown)$"
+        },
+        "post_treatment_time_units": {
+            "title": "Post-treatment time units",
+            "description": "The units of time that elapsed past the point when biosamples were washed from the treatment solution.",
+            "comment": "Use in conjunction with post_treatment_time and treatment to specify time elapsed post-treatment. post_treatment_time_units specification requires non-empty treatments list and post_treatment_time specification.",
+            "type": "string",
+            "enum": [
+                "minute",
+                "hour",
+                "day",
+                "week",
+                "month"
+            ]
+        },
+        "treatment_type": {
+            "title": "Treatment type",
+            "description": "The classification of the treatment.",
+            "type": "string",
+            "enum": [
+                "chemical",
+                "protein"
+            ]
+        },
+        "treatment_term_id": {
+            "@type": "@id",
+            "title": "Biosample treatment term ID",
+            "description": "Ontology identifier describing a component in the treatment.",
+            "type": "string",
+            "pattern": "^((CHEBI:[0-9]{1,7})|(UniProtKB:[A-Z0-9]{6})|(NTR:[0-9]{2,8}))$"
+        },
+        "treatment_term_name": {
+            "title": "Biosample treatment",
+            "description": "Ontology term describing a component in the treatment.",
```

Comment:
add a comment providing example of ontological terms for chemical and protein treatments